crawl (1:4.0.0beta24-6.kon.1b) local; urgency=medium

  * Local version!
  * Disable __attribute__((packed)) on Alpha, avoiding old GCC bug.

 -- Kalle Olavi Niemitalo <kon@iki.fi>  Sat,  8 Mar 2003 18:59:29 +0200

diff -ru crawl-4.0.0beta24-6/source/mon-util.h crawl-4.0.0beta24-6.kon.1b/source/mon-util.h
--- crawl-4.0.0beta24-6/source/mon-util.h	Wed May 29 23:45:56 2002
+++ crawl-4.0.0beta24-6.kon.1b/source/mon-util.h	Sat Mar  8 18:54:09 2003
@@ -18,7 +18,7 @@
 
 // ****remember***** must make an hardcopy of this sometime
 
-#if defined(macintosh) || defined(__IBMCPP__) || defined(SOLARIS) || defined(__BCPLUSPLUS__)
+#if defined(macintosh) || defined(__IBMCPP__) || defined(SOLARIS) || defined(__BCPLUSPLUS__) || defined(__alpha__)
 #define PACKED
 #else
 #define PACKED __attribute__ ((packed))
